### ISSUE TITLE
SNOW-904804: Cannot install snowflake-cli-labs==1.0.1 due to missing snowflake-connector-python-nightly version

### ISSUE THREAD

**Issue by pelletgu** [Snowflake-Labs/snowcli#344]
### Python version

python 3.11.4

### Platform

Linux-5.10.16.3-microsoft-standard-WSL2-x86_64-with-glibc2.31

### What happened

Hi,

It looks like I cannot import `snowflake-cli-labs==1.0.1` anymore because of a non existing `snowflake-connector-python-nightly` version

### Console output

```Shell
`ERROR: Could not find a version that satisfies the requirement snowflake-connector-python-nightly[secure-local-storage]==2023.6.24 (from snowflake-cli-labs) (from versions: 2023.6.28, 2023.6.29, 2023.7.6, 2023.7.8, 2023.7.11, 2023.7.12, 2023.7.13, 2023.7.14, 2023.7.15, 2023.7.18, 2023.7.20, 2023.7.21, 2023.7.28, 2023.7.29, 2023.8.1, 2023.8.5, 2023.8.10, 2023.8.15, 2023.8.24, 2023.8.25, 2023.8.29, 2023.8.30, 2023.8.31)
ERROR: No matching distribution found for snowflake-connector-python-nightly[secure-local-storage]==2023.6.24`
```

### How to reproduce

Create a `requirement.txt` and put `snowflake-cli-labs==1.0.1` then run `pip install -r requirement.txt` in an environnement with the 3.11.4 python version

**Comment by sfc-gh-pjob** [comment]
We prepared release candidate of Snowflake CLI 1.1.0 (1.1.0-rc1). This version will have fixed this issue.

Currently it can be installed via pip:
```
pip install git+https://github.com/Snowflake-Labs/snowcli.git@v1.1.0-rc1
```

We're planning to do an official release next week.

**Comment by sfc-gh-pjob** [comment]
We've just released official `1.1.0` version. It contains the fix, so I'm closing this issue. If sth connected to this issue is still wrong, please let us know.

---

Release notes: https://github.com/Snowflake-Labs/snowcli/releases/tag/v1.1.0

You can install it the same way as always - using pip or brew.
```
pip install snowflake-cli-labs
```
or
```
brew tap Snowflake-Labs/snowcli
brew update
brew install snowcli
```
